code/modules/mob/living/simple_animal/hostile/bear.dm

```diff
@@ -99,8 +99,57 @@
 		to_chat(user, "<span class='info'>You strap the armor plating to [A] and sharpen [A.p_their()] claws with the nail filer. This was a great idea.</span>")
 		qdel(src)
 
+mob/living/simple_animal/hostile/bear/butter //The mighty companion to Cak. Several functions used from it.
+	name = "Terrygold"
+	icon_state = "butterbear"
+	icon_living = "butterbear"
+	icon_dead = "butterbear_dead"
+	desc = "I can't believe its not a bear!"
+	faction = list("neutral")
+	obj_damage = 0
+	melee_damage_lower = 5
+	melee_damage_upper = 10
```

Comment:
You, uh, MIGHT not want to give a creature with 60 health and the ability to regenerate 10 HP per tick and the ability to knock people down on a hit the ability to directly hurt people. Make its melee_damage_lower and melee_damage_upper stats 0 and buff its chance to knock people down on each hit to, say, 100% (but make it unable to "slip" prone people), though, and this should be fine.

Reply:
Also, keep in mind that butterbears will be able to wear bear armor like normal bears can (which I think should be left in as a feature) to gain incredible stat buffs:
```
		A.armored = TRUE
		A.maxHealth += 60
		A.health += 60
		A.armour_penetration += 20
		A.melee_damage_lower += 5
		A.melee_damage_upper += 5
```

Reply:
I think all your suggestions are pretty good ones; frankly I was just sort of designing something to be goofy and never gave it much in the way of balance implications. I dropped the melee damage down to 1, attacks by the butterbear now slip targets 100% of the time but don't affect prone targets, gave it an object damage of 11 so it doesn't get trapped, and threw the russian faction in there.

I think the bear armor thing is pretty funny, so I think it deserves to stay in. I intended them to more of just be annoying and not really much of combatants, but if someone wants to go through all that work to try and give them some power I think thats fair.